pkg/kudoctl/cmd/install/install.go

```diff
@@ -181,59 +207,95 @@ func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.C
 			if helpers.AskForConfirmation() {
 				// If Instance is a dependency we need to make sure installSingleInstanceToCluster is aware of it.
 				// By having the previous string set we can make this distinction.
-				err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
+				err := installSingleInstanceToCluster(name, previous, bundlePath, kc)
 				if err != nil {
 					return errors.Wrap(err, "installing single Instance")
 				}
 			}
 		} else {
-			err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
+			err := installSingleInstanceToCluster(name, previous, bundlePath, kc)
 			if err != nil {
 				return errors.Wrap(err, "installing single Instance")
 			}
 		}
 
 	}
-
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkToCluster installs a given Framework to the cluster
-func installSingleFrameworkToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkYaml, err := gc.GetFrameworkYaml(name, path)
+func installSingleFrameworkToCluster(name, path string, kc *kudo.Client) error {
+	frameworkPath := path + "/" + name + "-framework.yaml"
```

Comment:
Same feedback as the other format, can we turn this into a format string const and fmt.Sprintf it?